packages/salesforce/test/http.test.js

```diff
@@ -0,0 +1,156 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import * as http from '../src/http';
+
+describe('http', () => {
+  describe('request', () => {
+    let spy;
+    const fakeConnection = {
+      request: () => Promise.resolve({ label: 'Account' }),
+    };
+    let state = { connection: fakeConnection };
+
+    beforeEach(() => {
+      spy = sinon.spy(fakeConnection, 'request');
+    });
+
+    afterEach(() => {
+      spy.restore(); // This restores all Sinon stubs and spies
+    });
+
+    it('should use relative path from root', done => {
+      http
+        .request('/services/data/v46.0/sobjects/Account/describe')(state)
+        .then(({ data }) => {
+          expect(spy.args[0]).to.eql([
```

Comment:
This test will need rewriting after the merge.

I would rather avoid using sinon if we can. That might get easier with `fakeConnection` in the new structure.

Reply:
I have removed `sinon` and refactor unit tests, We should be good now